Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.h

```diff
@@ -116,6 +116,11 @@ class MediaPlayerPrivateGStreamerMSE : public MediaPlayerPrivateGStreamer {
 
     bool m_isWaitingForPreroll = true;
     MediaPlayer::ReadyState m_mediaSourceReadyState = MediaPlayer::ReadyState::HaveNothing;
+
+    static unsigned mediaMaxWidth;
+    static unsigned mediaMaxHeight;
+    static unsigned mediaMaxFrameRate;
+    static bool isVideoDecoderLimitDefaultParsed;
```

Comment:
Minor improvement: you could declare these inside `MediaPlayerPrivateGStreamerMSE::supportsType` instead, to give them the smallest scope necessary and avoid the need to initialize them in a different place from where they are declared.